task2-bash/run_program.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+COUNT=0
```

Comment:
You can avoid double variable initialization by using return statement in run_program()

Reply:
Resolved